spec/messenger_spec.rb

```diff
@@ -25,6 +27,8 @@
   end
 
   describe '#deliver' do
+    before { Timecop.freeze(sent_at) }
```

Comment:
Timecop is a perfectly valid dependency to add for testing but you can also do

```
time_now = Time.now
allow(Time).to receive(:now).and_return(time_now)
```

Reply:
reasonable; though the one advantage with Timecop is that it will deal with methods that aren't just `now`.